Sources/Shapes.swift

```diff
@@ -201,17 +201,57 @@ public extension Path {
 }
 
 public extension Mesh {
+    /// A choice of the face directions that Euclid generates for polygons.
+    ///
+    /// ## Topics
+    ///
+    /// ### Faces
+    ///
+    /// - ``Faces/default``
+    /// - ``Faces/front``
+    /// - ``Faces/back``
+    /// - ``Faces/frontAndBack``
+    ///
+    /// ### Comparing Faces
+    ///
+    /// - ``Faces/!=(_:_:)``
+    ///
     enum Faces {
+        /// Render the front faces of the polygon.
```

Comment:
@nicklockwood Not time critical, but when you get a chance - would you look over this addition and makes sure it lines up with how you'd like it presented? I read through the code a bit to (try and) verify my understanding of what it represented, but I'm not 100% confident here.

Reply:
"Render" is probably not the right terminology here, because this relates to mesh generation (i.e. which faces are generated) rather than rendering. I wouldn't worry about it for now though - I'll do a pass correcting any such errors prior to merging.